ansible/roles/package-upgrade/tasks/partials/yum.yml

```diff
@@ -5,4 +5,6 @@
 #
 
   - name: upgrade installed packages
-    yum: name=* state=latest
+    yum: name=* state=latest use_backend=yum
+# XXX might have needed `use_backend=yum` at end of last line, but prob
+# not, was a workaround for a fixed path problem
```

Comment:
```suggestion
# If difficulty recognizing yum is encountered, see https://github.com/ansible/ansible/pull/69484, it is possible
# to apply that patch manually to a local install of ansible (if necessary).
```

Reply:
"use_backend=yum" is the workaround for not being able to detect `yum` on IBM i (fixed by https://github.com/ansible/ansible/pull/69484 but I've no idea when that will end up in an ansible release). If the ansible patch is manually applied then the `use_backend` can be dropped but it would be nicer not to require maintainers to have patched versions of ansible in order to run the scripts. 

So the remaining question is whether forcing the `yum` backend will cause issues for any of the other platforms. I think the answer to that is "no", at least as far as I am aware Centos 7 still defaults to `yum` (Centos 8 defaults to `dnf` but we don't have that in our CI). 

If we wanted to be ultra-cautious we could have a copy of the task with the `use_backend=yum` that is only run on IBM i (elsewhere we do this for installing packages).